CMakeLists.txt

```diff
@@ -8,15 +8,14 @@ cmake_minimum_required(VERSION 3.15)
 cmake_policy(SET CMP0079 NEW)
 cmake_policy(SET CMP0091 NEW)
 
-set(version "1.4.0" CACHE STRING "Project version")
-
-string(REGEX MATCH "[0-9]+\\.[0-9]+\\.[0-9]" version_short ${version})
+set(BUILD_NUMBER "0" CACHE STRING "Build number")
+set(VERSION_STRING "1.4.0" CACHE STRING "Project version (string)")
```

Comment:
Why did you change the name of this option? I would keep it the way it was as I don't see a need to change it :thinking:

Reply:
You mean `version`? I originally changed the name because when first working on this I created another option to specify the short version.

At the end I left a single option which specifies the full version.

Reply:
> You mean version?

Yes I do.

> At the end I left a single option which specifies the full version.

So this change is not needed after all? In that case I'd vote for reverting it :point_up:

Reply:
I agree.